billing-db/sprocs/get_tenant_bill.sql

```diff
@@ -20,7 +20,7 @@ CREATE TEMPORARY TABLE billable_resources
 );
 
 -- The billable_by_component table needs creating before running this stored function. This is so we can preserve the contents of this table for audit/debug purposes.
-CREATE TEMPORARY TABLE billable_by_component
+CREATE TABLE IF NOT EXISTS billable_by_component
```

Comment:
The billable_by_component and billable_resources tables need to be created as temporary tables (part of the new billing design). This code won't work otherwise - bills will get incorrectly calculated if more than one thread is running (e.g. more than one tenant bill being calculated at once or if later the billing calculator is being used).

Reply:
Thanks, I'll fix that. I think it was just changed for debugging purposes.